include/singa/core/device.h

```diff
@@ -51,6 +51,17 @@ using std::vector;
 
 namespace singa {
 
+/// Method used by time profiling
+/// ProfilingMode == 0 (default) -> Chrono
+/// ProfilingMode == 1 -> Event (e.g. cudaEvent)
+enum ProfilingMode { useChrono, useEvent };
+
+/// Verbosity of the time profiling function:
+/// Verbosity == 0 (default) -> no logging
+/// Verbosity == 1 -> display forward and backward propagation time
+/// Verbosity == 2 -> display each operation time (OP_ID, op name, time)
+enum Verbosity { disableLog, forwardBackwardTime, eachOperation };
```

Comment:
I think verbosity is typically set as a number...
We can refer to other project's code on how to set verbosity, giving it a name or just using int.
logging has INFO, WARN, ERROR, which are self-explainable.

Reply:
https://www.tensorflow.org/api_docs/python/tf/autograph/set_verbosity
Tensorflow is using number, so I will use number for verbosity

Reply:
I have modified the code: used int for Verbosity